contrib/fstrm/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "fstrm"
+pkgver = "0.6.1"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = ["automake", "libtool", "pkgconf", "gmake"]
```

Comment:
```suggestion
hostmakedepends = ["automake", "slibtool", "pkgconf", "gmake"]
```

where it works we prefer to start with this now instead, seems to work here

(and doesn't for knot below; i checked)